honeybee_radiance_command/rfluxmtx.py

```diff
@@ -0,0 +1,139 @@
+"""rfluxmtx command"""
+
+from ._command import Command
+import honeybee_radiance_command._exception as exceptions
+import honeybee_radiance_command._typing as typing
+from .options.rfluxmtx import RfluxmtxOptions, RfluxmtxControlParameters
+
+
+# octree to octree and so on
+class Rfluxmtx(Command):
+    """
+    Rfluxmtx command.
+
+    Rfluxmtx computes the flux transfer matrices for a RADIANCE scene.
+
+    """
+
+    __slots__ = ('_input', "_sensors", "_sender", "_octree", "_receivers", "_system")
+
+    def __init__(self, options=None, output=None, sensors=None, sender=None, receivers=None, system=None,
+                 octree=None):
+        """Initialize Command."""
+        Command.__init__(self, output=output)
+        self.options = options
+        self.sensors = sensors
+        self.octree = octree
+        self.sender = sender
+        self.receivers = receivers
```

Comment:
This is up for discussion but I _feel_ since this refers to a single file we can call it `self.receiver` and clarify in the docstring.

Reply:
This refers to a single file that can have multiple receivers and the multiple receivers, would come in handy when we do view factor calculations. So, I'd suggest keeping it as receivers just to maintain consistency with the radiance functionality.